q.js

```diff
@@ -203,7 +212,16 @@ var nextTick =(function () {
             setTimeout(flush, 0);
         };
     }
-
+    // runs a task after all other tasks have been run
+    // this is useful for unhandled rejection tracking that needs to happen
+    // after all `then`d tasks have been run.
+    nextTick.runAfter = function(task){
```

Comment:
This is sometimes called an idle event, and it’s fine as long as it does not effect a resume. I’ll take it as-is.